src/lib.rs

```diff
@@ -6,43 +6,53 @@ use std::{fs, process};
 
 use anyhow::{bail, ensure, Context, Result};
 use notify::{RecommendedWatcher, RecursiveMode, Watcher};
-use structopt::StructOpt;
+use structopt::{lazy_static, StructOpt};
```

Comment:
lol yeah no you can't do that xD it's not part of the public API. It has been hidden intentionally: https://docs.rs/structopt/latest/src/structopt/lib.rs.html#1117-1119

```rust
/// **This is NOT PUBLIC API**.
#[doc(hidden)]
pub use lazy_static;
```

You need to add lazy_static to your dependencies explicitly.